.github/actions/discover-backend/action.yml

```
@@ -0,0 +1,290 @@
+          fi
+          
+          # Set outputs
+          echo "backend-found=$BACKEND_FOUND" >> $GITHUB_OUTPUT
+          echo "api-endpoint=$API_ENDPOINT" >> $GITHUB_OUTPUT
+          echo "stack-name=$STACK_NAME" >> $GITHUB_OUTPUT
+          echo "resolution-method=$RESOLUTION_METHOD" >> $GITHUB_OUTPUT
+          echo "fallback-used=$FALLBACK_USED" >> $GITHUB_OUTPUT
+          echo "connectivity-status=$CONNECTIVITY_STATUS" >> $GITHUB_OUTPUT
+          echo "response-time=$RESPONSE_TIME" >> $GITHUB_OUTPUT
+          echo "discovery-result=$DISCOVERY_RESULT" >> $GITHUB_OUTPUT
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)